.travis.yml

```diff
@@ -8,6 +8,7 @@ before_script:
   - swiftlint --strict
 matrix:
   include:
-    - env: NAME=Xcode DESTINATION="platform=iOS Simulator,name=iPhone 8"
+    - env: NAME=Xcode DESTINATION="platform=iOS Simulator,name=iPhone 8" SCHEME="Crossroad"
```

Comment:
I prefer to rename to `Crossroad-iOS`. I'll rename this after merging this.

Reply:
Ok, great! I will leave this one alone for you to change.